src/utils/web3.ts

```diff
@@ -49,6 +50,23 @@ export function accountTruncate(account: string): string {
   return truncated
 }
 
+export function getNetworkType(network: EthereumListsChain): string {
+  // HEADS UP! Hack for getting network's type main/test, without using
+  // .network field, which is innexistent on https://chainid.network/chains.json
+  // We hack in mainnet detection for moonriver.
+
+  if (
+    (!network.name.includes('Testnet') &&
+      !network.title?.includes('Testnet') &&
+      network.name !== 'Moonbase Alpha') ||
+    network.name === 'Moonriver'
```

Comment:
I don't think we need this check for 'Moonriver', it should label it as `mainnet` without this check